e3sm_diags/plot/utils.py

```diff
@@ -743,7 +743,7 @@ def _add_rmse_corr_text(
     fontdict = {"fontsize": fontsize}
 
     if left_text_pos is None:
-        left_text_pos = (0.6335, -0.0105)
+        left_text_pos = (0.6635, -0.0105)
```

Comment:
Aligns with https://github.com/E3SM-Project/e3sm_diags/blob/ca41b0e5d913610c88410928951f1ed11c75663f/e3sm_diags/plot/cartopy/lat_lon_plot.py#L223

Reply:
I suppose this will fix other 3 panel plots that uses the lat_lon_plot.py? because I do see same problems in other sets that has the string `RMSE` `CORR` mis-placed, some are over lapping with figures.

Reply:
Okay, I checked, `lat_lon`, `zonal_mean_2d/stratosphere`, `meridional_mean_2d`, `polar` will all be fixed.